index.html

```diff
@@ -32,76 +33,91 @@ <h2 id="about">about</h2>
 		<h2 id="projects">projects</h2>
 	
 		<article>
-			
 			<div class="textContent">
 				<div class="projectInfo">
-					<p class="projectTitle">project 1</p>
+					<a href="#">
+						<p class="projectTitle">project 1</p>
+					</a>
 					<p>08/09/21</p>
-					<a href="#"><i class="fab fa-github">    github</i></a>
+					<a href="#">
+						<i class="fab fa-github icon">    github</i>
+					</a>
 				</div>
 			
 				<div class="projectDescription">
 					<p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
 				</div>
 			</div>
 			
-			<img src="cloudOne.jpg" alt="Clouds in the sky">
-			
+				<img src="cloudOne.jpg" alt="Clouds in the sky">
 		</article>
 		
 		<article>
-			
 			<div class="textContent">
 				<div class="projectInfo">
-					<p class="projectTitle">project 1</p>
+					<a href="#">
+						<p class="projectTitle">project 2</p>
+					</a>
 					<p>08/09/21</p>
-					<a href="#"><i class="fab fa-github">    github</i></a>
+					<a href="#">
+						<i class="fab fa-github icon">    github</i>
+					</a>
 				</div>
 			
 				<div class="projectDescription">
 					<p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
 				</div>
 			</div>
 			
-			<img src="cloudOne.jpg" alt="Clouds in the sky">
-			
+				<img src="cloudTwo.jpg" alt="Clouds in the sky">
 		</article>
 		
 		<article>
-			
 			<div class="textContent">
 				<div class="projectInfo">
-					<p class="projectTitle">project 1</p>
+					
+					<a href="#">
+						<p class="projectTitle">project 3</p>
+					</a>
 					<p>08/09/21</p>
-					<a href="#"><i class="fab fa-github">    github</i></a>
+					<a href="#"><i class="fab fa-github icon">    github</i></a>
 				</div>
 			
 				<div class="projectDescription">
 					<p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
 				</div>
 			</div>
 			
-			<img src="cloudOne.jpg" alt="Clouds in the sky">
-			
+				<img src="cloudThree.jpg" alt="Clouds in the sky">
 		</article>
 		
 		<article>
-			
 			<div class="textContent">
 				<div class="projectInfo">
-					<p class="projectTitle">project 1</p>
+					<a href="#">
+						<p class="projectTitle">project 4</p>
+					</a>
 					<p>08/09/21</p>
-					<a href="#"><i class="fab fa-github">    github</i></a>
+					<a href="#" target="_blank"><i class="fab fa-github icon">    github</i></a>
 				</div>
 			
 				<div class="projectDescription">
 					<p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
 				</div>
 			</div>
 			
-			<img src="cloudOne.jpg" alt="Clouds in the sky">
-			
+				<img src="cloudFour.jpg" alt="Clouds in the sky">
 		</article>
 		
+		
+		<section class="foot">
+			<div class="footInfo">
+				<h2>get in touch:</h2>
+				<p class="email">kim.dobney@yahoo.co.uk</p>
```

Comment:
This could be an `<a>` that opens a new email addressed to you when clicked.
`<a href="mailto:kim.dobney@yahoo.co.uk" target="_blank" class="email">kim.dobney@yahoo.co.uk</a>`

Reply:
perfect i've added that in